src/kvstore/RocksEngineConfig.cpp

```diff
@@ -196,6 +199,10 @@ rocksdb::Status initRocksdbOptions(rocksdb::Options &baseOpts) {
     }
     baseOpts.table_factory.reset(NewBlockBasedTableFactory(bbtOpts));
     baseOpts.create_if_missing = true;
+    if (FLAGS_enable_prefix_bloom_filter) {
+        baseOpts.prefix_extractor.reset(rocksdb::NewFixedPrefixTransform(
+                sizeof(PartitionID) + sizeof(VertexID)));
```

Comment:
I prefer to use rocksdb.FixedPrefix.16, for most of the case prefix is used by partid+vid+typeid.

Reply:
```
kvstore::ResultCode QueryVertexPropsProcessor::collectVertexProps(
                            PartitionID partId,
                            VertexID vId,
                            std::vector<cpp2::TagData> &tds) {
    auto prefix = NebulaKeyUtils::vertexPrefix(partId, vId);
```
If the prefix is 16 bytes, QueryVertexPropsProcessor can't use the filter? since it use partId+vId as the seek key

Reply:
Exactly, we support scan all tags for one vertex.

Reply:
QueryVertexPropsProcessor::collectVertexProps is only used in `fetch prop on <ids> <no yield>`. For most of case, it is used for debug and test.

Reply:
We can't have this assumption

Reply:
If the query condition is (partid+vid+typeid), it can also use (partid+vid) as the prefix filter. do you think 12bytes is good enough? @xuguruogu

Reply:
> If the query condition is (partid+vid+typeid), it can also use (partid+vid) as the prefix filter. do you think 12bytes is good enough? @xuguruogu

This miss rate may be high if multi edge/tag type associated with the vid. So why you introduce the prefix bloom filter.

Reply:
IMO, (partid+vid) is good enough, or do you think there is a need to make this configuable